.github/workflows/format_check.yml

```diff
@@ -0,0 +1,59 @@
+name: Check for formatting errors on head ref
```

Comment:
Not a blocking comment but in the near future we're going to want to subsume a larger class of checks than just formatting checks in this workflow, so its name should reflect that.

Reply:
I'd prefer keeping this name as it is as we don't know yet what's going to be in this "larger class of checks" and we might want to split them in multiple workflows